tests/benchq/compilation/test_rbs_hyperparam_tuning.py

```diff
@@ -0,0 +1,291 @@
+import os
+import random
+import string
+
+import pytest
+from orquestra.quantum.circuits import CNOT, Circuit, H
+
+from benchq.compilation import transpile_to_native_gates
+from benchq.compilation.rbs_hyperparam_tuning import (
+    create_estimated_rbs_time_objective_fn,
+    create_space_time_objective_fn,
+    estimated_time_cost_from_rbs,
+    get_optimal_hyperparams_for_estimated_rbs_time,
+    get_optimal_hyperparams_for_space,
+    get_optimal_hyperparams_for_space_and_time,
+    get_optimal_hyperparams_for_time,
+    space_time_cost_from_rbs,
+)
+
+
+@pytest.fixture()
+def large_circuit():
+    size = 200
+    circ = Circuit(
+        [
+            H(0),
+            *[CNOT(j, i) for i in range(1, size) for j in range(0, size)],
+        ]
+    )
+    return transpile_to_native_gates(circ)
+
+
+@pytest.fixture()
+def small_circuit():
+    circ = Circuit([H(0), CNOT(0, 1)])
+    return transpile_to_native_gates(circ)
+
+
+SKIP_SLOW = pytest.mark.skipif(
```

Comment:
This is a good idea! We should use this other places in the code!

Reply:
Maybe it should be the opposite where you have to set the environment variable to just do the fast benchmarks.

Reply:
We could even integrate it into the `make test` command!

Reply:
We [do use this in other places](https://github.com/zapatacomputing/benchq/blob/main/benchmarks/test_get_qsp_program.py#L18) in the code! I [stole it](https://starecat.com/content/wp-content/uploads/designers-you-stole-my-idea-programmers-man-i-stole-your-code-its-not-my-code.jpg) 😄

Reply:
Ok! I'll use it in other places of the code too!